public/app/features/correlations/CorrelationsPage.tsx

```diff
@@ -279,7 +289,7 @@ const InfoCell = memo(
     const readOnly = props.row.original.provisioned;
 
     if (readOnly) {
-      return <Badge text="Read only" color="purple" className={noWrap} />;
+      return <Badge text={t('correlations.badge.text', 'Read only')} color="purple" className={noWrap} />;
```

Comment:
In this case the key should probably indicate the message on the badge. I think something like `correlations.list.read-only` would be more meaningful. We could also use `correlations.list.` to translate other labels in the list (deleting correlations, column names, etc.)